packages/components/src/Avatar/Avatar.spec.tsx

```diff
@@ -23,4 +23,16 @@ describe("<Avatar />", () => {
       expect(screen.getByText("JWAVLN")).toBeInTheDocument()
     })
   })
+
+  describe("alt text", () => {
```

Comment:
I think there needs to be some extra test cases. I see 4 different places you've changed but only have 2 tests :)

Reply:
I've built the tests out more. There's 3 possible renders:
* img src supplied
* full name provided with initials feature turned on
* fall back icon with no img or initials

So for each of these I've made sure it:
* uses alt text as the top priority
* uses full name in lieu of alt
* hides the elements if neither is provided